src/styles/styles/_card.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: $cardWidth;
+  height: 100%;
+  border: 1px solid $grayAccent;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__image {
+    height: 134px;
+    width: 160px;
+    align-self: center;
+  }
+  
+  &__label {
+    padding-top: 40px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $mainColor;
+  }
+  
+  &__code-product {
+    font-size: 10px;
+    line-height: 14px;
+    padding-top: 4px;
+    color: $color-secondary;
+  }
+  &__price {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $mainColor;
+    align-self: flex-end;
+    
+    &-container {
+      display: flex;
+      align-items: center;
+      justify-content: space-between;
+      padding-top: 24px;
+      padding-bottom: 16px;
+    }
+  
+    &-label {
+      font-size: 12px;
+      line-height: 18px;
+      color: $color-secondary;
+    }
+  }
+  
+  &__button {
+    margin: 0 auto;
+    display: flex;
+    color: $whiteAccent;
+    width: 166px;
+    height: 40px;
+    background-color: $blueAccent;
+    border-radius: 5px;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-decoration: none;
+    justify-content: center;
+    align-items: center;
+    text-transform: uppercase;
+  }
+
+  &__button:hover {
+    border: 1px solid $blueAccent;
+    background-color: $whiteAccent;
+    border-radius: 5px;
+    color: $blueAccent;
+  }
```

Comment:
You can nest styles for hover to &__button selector
```
&__button {
  &:hover { ...

Reply:
not fixed 
<img width="424" alt="image" src="https://user-images.githubusercontent.com/53611910/226131412-1619fbc1-04b3-4ea6-93b9-f6b22d9a6e0d.png">